plugins/Marketplace/vue/src/PluginList/DownloadButton.vue

```diff
@@ -0,0 +1,53 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <template
+    v-if="plugin.missingRequirements.length === 0 && plugin.isDownloadable && !isAutoUpdatePossible"
+  >
+    <span onclick="$(this).css('display', 'none')">
+      <template v-if="showOr">{{}} {{ translate('General_Or') }} {{}}</template>
```

Comment:
I presume the empty `{{}}` are here to keep the space around the translated word 'or'? Can we use non-breakable spaces instead or at least add a comment to keep these? When I look at it in 6 months I will be scratching my head what it means again.

Reply:
Yes, the "common" pattern of forcing whitespace, a different variant being `{{ ' ' }}{{ ...`. But one of those `&nbsp;` won't hurt, so let's go with that.